### PR TITLE
Removes the singularity scare hallucination

### LINKED ISSUE #181: SOMETHING to do with Mindbreaker Toxin crashes Dreamseeker.
I don't know exactly what causes this. If nobody else can be fucked, I'll try to do some testing on my own server to see what it is. All I know so far is A. It's caused by hallucinations/mindbreaker and B. It's not instant, nor is it on a fixed delay. Speculation here, I'm pretty sure it's caused by a specific hallucination that somehow crashes dreamseeker.

### LINKED ISSUE #348: mindbreaker toxin crashing the client
In-Game report from zadingtons:
Revision info: Server revision 288eca3d7d0346521d83a97a033471c60f72f17a compiled on: 2017-02-23

>General description of the issue
The captain gave me 40 units of mindbreaker toxin against my will which lead to multiple instances of hallucination (I can recall 3 happening at once I think). On random instances, which seem to be the same type of hallucination occuring, would the client actually crash. I recall something being changed/removed due to crashes relating to hallucination before but something else seems to be causing it now.

>What you expected to happen
for the game not to crash

>What actually happened
the game crashed

>Steps to reproduce if possible
One 40u patch of Mindbreaker toxin and a game of chance to see what hallucination is exactly causing the crashing.

### PR DESCRIPTION
**Pull request by onein50million**
:cl: Spyridon10
fix: Removes the singularity hallucination, as it was causing crashes.
/:cl:

I couldn't figure out why, but for some reason the singularity scare hallucination occasionally crashes Dream Seeker. 
fixes #348 
fixes #181

### CODE REVIEW

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -218,6 +218,8 @@ Gunshots/explosions/opening doors/less rare audio (done)
 /obj/effect/hallucination/simple/clown/scary
 	image_state = "scary_clown"
 
+//Singularity scare causes crashes for some reason
```
Comment (ghost) [inline code/modules/flufftext/Hallucination.dm:221]: don't comment this out, just remove it from the `major` list

**Review by ghost — DISMISSED** [review]

File: code/modules/flufftext/Hallucination.dm [hunk 2]
```diff
@@ -237,7 +239,8 @@ Gunshots/explosions/opening doors/less rare audio (done)
 		s.Eat()
 	qdel(s)
 
-/obj/effect/hallucination/simple/singularity
+/obj/effect/hallucination/simple/
```
Comment (tortellinitony) [inline code/modules/flufftext/Hallucination.dm]: Why add this?

**Review by tortellinitony — CHANGES_REQUESTED** [review]

File: code/modules/flufftext/Hallucination.dm [hunk 3]
```diff
@@ -551,8 +554,8 @@ var/list/non_fakeattack_weapons = list(/obj/item/weapon/gun/ballistic, /obj/item
 	switch(hal_type)
 		if("xeno")
 			new /obj/effect/hallucination/xeno_attack(src.loc,src)
-		if("singulo")
-			new /obj/effect/hallucination/singularity_scare(src.loc,src)
+//		if("singulo")
```
Comment (tortellinitony) [inline code/modules/flufftext/Hallucination.dm]: Delete this instead of commenting it out

**Review by tortellinitony — CHANGES_REQUESTED** [review]

File: code/modules/flufftext/Hallucination.dm (hunk 3 above)
Comment (ghost) [reply code/modules/flufftext/Hallucination.dm]: Just remove it from the `major` list

File: code/modules/flufftext/Hallucination.dm (hunk 2 above)
Comment (onein50million) [reply code/modules/flufftext/Hallucination.dm]: I must have deleted that by accident. I'll fix it

**Review by tortellinitony — APPROVED** [review]
Looks good

**Comment by KayeArray** [comment]
we should probably fix this or replace it with something else at some point, hell maybe even a hallucination overhaul

**Pull request merged**